index.html

```diff
@@ -26,75 +27,121 @@
         <!--[if lt IE 8]>
             <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
```

Comment:
Indentation is messed up in the whole file. IMHO, it should be 2 or 4 spaces consistent for every parent-child relationship in DOM elements.